IDE/MonoDevelop/MonoDevelop.MonoGame.Android/templates/MonoGameAndroidProject.xpt.xml

```diff
@@ -22,17 +22,15 @@
 				AndroidApplication="true"
 				AndroidResgenFile="Resources/Resource.designer.cs"
 				AndroidResgenClass="Resource"
-				TargetFrameworkVersion="MonoAndroid,Version=v4.4" />
+				TargetFrameworkVersion="MonoAndroid,Version=v5.0" />
 			<References>
 				<Reference type="Gac" refto="System, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e" />
 				<Reference type="Gac" refto="System.Xml, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e" />
 				<Reference type="Gac" refto="System.Core, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e" />				
 				<Reference type="Gac" refto="Mono.Android" />
-				<Reference type="Gac" refto="OpenTK" />
+				<Reference type="Gac" refto="OpenTK-1.0" />
```

Comment:
We've been building iOS and Android frameworks against OpenTK-1.0 for a while now. The templates were out of date.

https://github.com/mono/MonoGame/blob/develop/Build/Projects/FrameworkReferences.definition#L8
https://github.com/mono/MonoGame/blob/develop/Build/Projects/FrameworkReferences.definition#L47